### DISCUSSION TITLE
Master server stuck in starting

### DISCUSSION THREAD

**Discussion by kvaster** [seaweedfs/seaweedfs#5438] [Q&A]
I'm running seaweedfs in kubernetes (my own helm chart based on official) and from time time I have problems with starting master server.

There three master servers. Sometimes after rebooting/restarting one of them it is stuck in starting. It can see other master servers, but it never finish starting. And it can't join even if I will restart/kill it once more. There is only one way to have all three master servers working in such situation - I need to restart all of them.

What can I do to investigate this problem ?

**Comment by chrislusf** [comment]
Try to run masters with "-v=4".

**Comment by n9** [comment]
I just want to confirm this issue. It happens to me too. (And I also need to restart all 3 masters.)

**Comment by kmlebedev** [comment]
> I just want to confirm this issue. It happens to me too. (And I also need to restart all 3 masters.)

are you using the -raftHashicorp option?
And it would be useful to give your example of running weed cmd with all the arguments.

**Reply by n9** [reply]
> are you using the -raftHashicorp option?

Not yet. But I will try.

> And it would be useful to give your example of running weed cmd with all the arguments.

```yaml
command:
  - /usr/bin/weed
  - '-logtostderr=true'
  - '-v=1'
  - master
  - '-ip=$(POD_NAME).master.seaweedfs'
  - '-ip.bind=0.0.0.0'
  - '-port=9333'
  - >-
    -peers=master-0.master.seaweedfs:9333,master-1.master.seaweedfs:9333,master-2.master.seaweedfs:9333
  - '-mdir=/data'
  - '-volumeSizeLimitMB=1024'
  - '-volumePreallocate'
  - '-defaultReplication=001'
  - '-metricsPort=9091'
```

and volume (just FYI):

```yaml
command:
  - /usr/bin/weed
  - '-logtostderr=true'
  - '-v=1'
  - volume
  - '-ip=$(POD_NAME).volume.seaweedfs'
  - '-ip.bind=0.0.0.0'
  - '-publicUrl=$(POD_NAME)-seaweedfs.example.com'
  - '-port=8080'
  - >-
    -mserver=master-0.master.seaweedfs:9333,master-1.master.seaweedfs:9333,master-2.master.seaweedfs:9333
  - '-dir=/data'
  - '-max=0'
  - '-readMode=proxy'
  - '-minFreeSpacePercent=7'
  - '-compactionMBps=50'
  - '-metricsPort=9091'
```

**Comment by kvaster** [comment]
I've launched all my clusters with `-v4` and waiting to reproduce the problem.
